docs/modules/operation/pages/user-management/introduction.adoc

```diff
@@ -39,6 +39,9 @@ When enabled, the `Data Choices` module collects the following anonymous statist
 * OS Architecture
```

Comment:
```suggestion
* OS architecture
```

Reply:
Just noticed that these shouldn't be capitalized. Could you also update for lines 37 and 38? "Release" and "Version" should be lower case.